dissect/target/plugins/os/windows/network.py

```diff
@@ -322,21 +339,17 @@ def _extract_network_device_config(self, interface_id: str) -> list[dict[str, se
         """Extract network device configuration from the given interface_id for all ControlSets on the system."""
 
         dhcp_config = {
```

Comment:
For the sake of consistency, can you order these the same as the record definition?